app/routes/articles/components/ArticleEditor.tsx

```diff
@@ -229,9 +229,9 @@ const ArticleEditor = () => {
               id="article-title"
             />
             <Field
-              placeholder="Skriv artikkelen din her ..."
+              placeholder="Skriv tråden din her"
               name="content"
-              label="Innhold"
+              label="Beskrivelse"
```

Comment:
Is this intentional? I'd imagine the old text is more apt for editing articles, which I assume it is still being used for?

If articles and forums are both going to be using this component it might be better to move it to `app/components/...` and allow this sorta stuff to be passed as a prop to suit both uses